rebase-blogpost/README.md

```diff
@@ -4,22 +4,23 @@ Let's first consider the basic example where you want to integrate a feature bra
 
 ![Example of merging](merge.gif)
 
-Alternatively, we could rebase before merging. The commits are removed and feature is reset to master, after which the commits are re-applied on top of feature. The diffs of these re-applied commits are usually identical to their original counterparts, but they have different parent commits, and hence different SHA-1 keys.
+Alternatively, we could rebase before merging. The commits are removed and the `feature` branch is reset to `master`, after which the commits are re-applied on top of `feature`. The diffs of these re-applied commits are usually identical to their original counterparts, but they have *different* parent commits, and hence different SHA-1 keys.
 
 ![Example of rebasing](rebase.gif)
 
-We've now changed the base commit of feature from `b` to `c`, literally re-basing it.
-Merging feature to master is now a fast-forward merge, because all commits on feature are direct descendants of master.
+We have now changed the base commit of `feature` from `b` to `c`, literally re-basing it.
+Merging `feature` to `master` is now a fast-forward merge, because all commits on `feature` are direct descendants of `master`.
 
 ![Example of fast forward merge](rebase-ff.gif)
 
 Compared to the merge approach, the resulting history is linear with no divergent branches. The improved readability is the reason many people prefer to rebase their branches before merging.
+> Is it? How do you know?
```

Comment:
Good point. I could rephrase it to say something like

"The improved readability was the reason I used to prefer rebasing branches before merging, and I expect this to be true for other developers as well."